src/Tests/dotnet.Tests/WindowsInstallerTests.cs

```diff
@@ -0,0 +1,218 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.ComponentModel;
+using System.IO;
+using System.IO.Pipes;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Runtime.Versioning;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.DotNet.Installer.Windows;
+using Microsoft.DotNet.Installer.Windows.Security;
+using Microsoft.NET.TestFramework;
+using Xunit;
+
+namespace Microsoft.DotNet.Tests
+{
+    [SupportedOSPlatform("windows")]
+    public class WindowsInstallerTests
+    {
+        private static string s_testDataPath = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "TestData");
```

Comment:
So you're laying down a test data dir next to the test assembly? Why can't we put this in the normal test assets location?

Reply:
Can we tell the package reference to restore into a different location?

Reply:
I chatted with Daniel, the package carries the assets as content files so it's expected to land in the the output folder. Our other option would be to add a target and use the download package feature, then unzip the nupkg and copy the contents over to a different location